test/ContentValidationListenerTest.php

```diff
@@ -1992,14 +1992,6 @@ public function testSaveUnknownDataWhenEmptyInputFilter()
     }
 
     /**
-     * @depends testFailsValidationOfPartialSetsForPatchRequestsForCollectionThatIncludeUnknownInputs
```

Comment:
In checking through the tests, I think this behavior is already tested with:

- testReturnsApiProblemResponseForCollectionIfAnyFieldsAreInvalid
- testValidatePostedCollectionsAndAllowedOnlyFieldsFromFilterReturnsApiProblemWithUnrecognizedFields
- testReportsValidationFailureForPostedCollection
- testIndicatesInvalidPostedEntityWhenCollectionIsPossibleForService
- testFailsValidationOfPartialSetsForPatchRequestsThatIncludeUnknownInputs

as each demonstrates the return of an `ApiProblem` for invalid collections; the latter is also specifically targeting the unknown inputs situation.